types/types.go

```diff
@@ -167,12 +167,16 @@ type DiscordOutputConfig struct {
 }
 
 type AlertmanagerOutputConfig struct {
-	HostPort        string
-	MinimumPriority string
-	CheckCert       bool
-	MutualTLS       bool
-	Endpoint        string
-	ExpiresAfter    int
+	HostPort             string
+	MinimumPriority      string
+	CheckCert            bool
+	MutualTLS            bool
+	Endpoint             string
+	ExpiresAfter         int
+	ExtraLabels          string
+	ExtraLabelsList      []string
```

Comment:
Why are you creating a slice and not a map[string]string?

Reply:
the parsing should be done at the client creation too

Reply:
I wanted to have constancy with Prometheus config and Loki one's.
I also didn't see there was already everything to parse the map from env vars.

Reply:
I guess it's cleaner now :D